cmd/init.go

```diff
@@ -154,6 +156,43 @@ func Initialize(ctx context.Context, providers []string) error {
 	return nil
 }
 
+// ParseProviderCLIArg parses the CLI arg passed to `cloudquery init <provider>`
+func parseProviderCLIArg(providerCLIArg string) (org string, name string, version string, err error) {
```

Comment:
See original in https://github.com/cloudquery/cloudquery/pull/909/files#diff-3064fe03f4e76c66e781135a4e8b2c0165c5edb182bea64248daa68da95d6a10L37

I split the conditions in the original to make it easier to know the different formats and expected results